src/rules/no_deprecated_deno_api.rs

```diff
@@ -124,6 +125,17 @@ impl DeprecatedApi {
       Replacement::NameAndUrl(name, url) => {
         format!("Use `{}` from {} instead", name, url)
       }
+      Replacement::NameAndUrls(name_and_urls) => {
+        let mut hint = String::from("Use ");
+        for (i, (name, url)) in name_and_urls.into_iter().enumerate() {
+          if i != 0 {
+            hint.push_str(" and ");
+          }
+          hint.push_str(&format!("`{}` from {}", name, url));
+        }
+        hint.push_str(" instead");
+        hint
+      }
```

Comment:
Can we get a unit test for the method `hint`? It would be great to confirm what hints are generated for complicated cases.

Reply:
Added unit test [68b0003](https://github.com/denoland/deno_lint/pull/1222/commits/68b00030dd7e11486b7b7d9fa73e0e531ec42efa)

Reply:
Thanks!

I'm a bit worried that putting all the information in one line might look overwhelming to users. 


![image](https://github.com/denoland/deno_lint/assets/23649474/ebefb295-7056-49b6-a464-85a477d7674f)

One idea that came up to my mind is to separate hints into multiple lines, where each line contains just one reference to alternative API. Something like this:

```
Use the following APIs instead
  - `WritableStream` from https://deno.land/api?s=WritableStream
  - `ReadableStream.from` from https://deno.land/api?s=ReadableStream#variable_ReadableStream
  - `ReadableStream.pipeTo` from https://deno.land/api?s=ReadableStream#method_pipeTo_4
```

But I'm not really sure if this is doable with the current mechanism of showing diagnostics that depends on `miette` crate.

What do you think?

Reply:
@magurotuna 
Yes, such diagnostic results are indeed easy to understand.
However, I am not familiar with miette, so I don't know if it can output such diagnostic results
Also, since this is a sample implementation, I wonder if I need to work that hard
How about drafting a separate issue from this PR and improving it?
I would like to limit the scope of this issue to fixing the wrong message for now.

Reply:
That makes sense, we can do further improvement in a follow-up PR. Let's land it as-is then 👍